station-boards-injector/pom.xml

```diff
@@ -30,21 +30,9 @@
   <version>1.0-SNAPSHOT</version>
 
   <properties>
-    <vertx.verticle>stream.InjectVerticle</vertx.verticle>
+    <vertx.verticle>stream.DelayedTrainsHttpVerticle</vertx.verticle>
```

Comment:
This needs to be changed when you'll split the project

Reply:
#12